back-end/controllers/main-controller.js

```diff
@@ -0,0 +1,99 @@
+const connection = require("../db") 
+
+const addProduct= (req,res)=>{
+const {storeId,productName,productDescripition,quantityPerUnit,unitPrice,availableProduct,picture}=req.body
+const data = [productName,productDescripition,quantityPerUnit,unitPrice,availableProduct,picture]
+const quary = `insert into products (productName,productDescripition,quantityPerUnit,unitPrice,availableProduct,picture)
```

Comment:
it should be `query`, not `quary`, also type the SQL code with its own conventions, for example, `INSERT INTO` instead of `insert into`